addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -372,7 +392,7 @@ var MassMailingFieldHtml = FieldHtml.extend({
         if (!odoo.debug) {
             $snippetsSideBar.find('.o_codeview_btn').hide();
         }
-        const $codeview = this.$content.parent().find('textarea.o_codeview');
+        const $codeview = this.$content.parent().parent().find('textarea.o_codeview');
```

Comment:
Aren't you better of just calling something like this.$el.find() instead of trying to guess how many times you need to move up the hierarchy before calling find?